python/hail/dataset.py

```diff
@@ -1375,7 +1375,7 @@ def downsample_variants(self, keep):
 
     @handle_py4j
     @requireTGenotype
-    def export_gen(self, output):
+    def export_gen(self, output, precision=4):
```

Comment:
I thought it was better to leave the default behavior as what it was in the past. 3 is probably better, but I definitely think 5 is overkill for most people's needs.